libs/vkd3d/device.c

```diff
@@ -1440,11 +1441,11 @@ static void vkd3d_physical_device_info_apply_workarounds(struct vkd3d_physical_d
                     info->properties2.properties.driverVersion >= VKD3D_DRIVER_VERSION_MAKE_NV(537, 0, 0) &&
                     info->properties2.properties.driverVersion < VKD3D_DRIVER_VERSION_MAKE_NV(537, 96, 0);
 
-            if (broken_version_linux || broken_version_windows)
+            if (broken_version_linux || broken_version_windows || (vkd3d_config_flags & VKD3D_CONFIG_FLAG_DISABLE_NV_DGCC))
```

Comment:
What is the bug exactly? Didn't the DGCC bugs get fixed? Or did it regress again?

Reply:
Seems to be happening only on Halo infinite: 
https://forums.developer.nvidia.com/t/regression-550-54-14-halo-infinite-permanently-hangs/284441